src/main/java/io/warp10/ext/interpolation/InterpolationWarpScriptExtension.java

```diff
@@ -37,6 +37,9 @@ public class InterpolationWarpScriptExtension extends WarpScriptExtension {
   public static final String INTERPOLATOR_1D_LINEAR = "INTERPOLATOR.1D.LINEAR";
   public static final String INTERPOLATOR_1D_SPLINE = "INTERPOLATOR.1D.SPLINE";
   public static final String INTERPOLATOR_1D_AKIMA = "INTERPOLATOR.1D.AKIMA";
+  public static final String INTERPOLATOR_TICK_LINEAR = "mapper.interpolator.linear";
+  public static final String INTERPOLATOR_TICK_SPLINE = "mapper.interpolator.spline";
+  public static final String INTERPOLATOR_TICK_AKIMA = "mapper.interpolator.akima";
```

Comment:
I already use the the other interpolator as mappers.
The interesting thing here is to define them from GTS, and that bucketizing is not mandatory (it is for INTERPOLATE)...
- GTS.INTERPOLATOR.SPLINE ?
- gts.to.mapper.spline ?

Reply:
or INTERPOLATE could take more optional arguments: a 1d interpolator on the tick domain, a list of addtitional 
ticks.